File: example/fizzbuzz.c
#include <stdio.h>

int main(void)
{
	for (int i = 1; i <= 50; i++)
	{
		if (i % (3 * 5) == 0)
		{
			printf("Fizz Buzz\n");
		}
		else if (i % 3 == 0)
		{
			printf("Fizz\n");
		}
		else if (i % 5 == 0)
		{
			printf("Buzz\n");
		}
		else
		{
			printf("%d\n", i);
		}
	}
}

File: example/fizzbuzz-rs/fizzbuzz.rs
fn main() {
	(1 ..= 50).for_each(|i| match (i % 3, i % 5) {
		(0, 0) => println!("Fizz Buzz"),
		(0, _) => println!("Fizz"),
		(_, 0) => println!("Buzz"),
		(_, _) => println!("{}", i),
	});
}
